service/flannel/error.go

```diff
@@ -1,19 +0,0 @@
-package flannel
```

Comment:
The flannel package got removed. Its logic is now being executed within the legacy resource.